### ISSUE TITLE
getTouchPointCount() doesn't count

### ISSUE THREAD

**Issue by Logickin-Lambda** [G3bE/raylib-zig#181]
Hi here, thank you for creating the binding so that building raylib application in zig has becoming very easy.

However, I have faced an issue about using touch screen:

My zenbook supports multitouch, so I have written a multitouch demo in zig based on [this example from the official raylib website](https://www.raylib.com/examples/core/loader.html?name=core_input_multitouch) which it works with my laptop , but no matter how, none of the fingers are responded on the screen. After I have traced the code, I just found that getTouchPointCount() never counts the number of fingers on the screen.

Here is my code:

```
const std = @import("std");
const rl = @import("raylib");
const gui = @import("raygui");

pub fn main() !void {
    // Initialization
    //--------------------------------------------------------------------------------------
    const screenWidth = 800;
    const screenHeight = 450;

    const MAX_TOUCH_POINTS = 10;

    rl.initWindow(screenWidth, screenHeight, "raylib-zig [core] example - basic window");
    defer rl.closeWindow(); // Close window and OpenGL context

    var touch_points: [MAX_TOUCH_POINTS]rl.Vector2 = std.mem.zeroes([10]rl.Vector2);

    rl.setTargetFPS(60); // Set our game to run at 60 frames-per-second
    //--------------------------------------------------------------------------------------

    // Main game loop
    while (!rl.windowShouldClose()) { // Detect window close button or ESC key
        // Update
        //----------------------------------------------------------------------------------
        // TODO: Update your variables here
        //----------------------------------------------------------------------------------
        var t_count = rl.getTouchPointCount();
        std.debug.print("Number of Touch Points: {d}\n", .{t_count});

        if (t_count > MAX_TOUCH_POINTS) t_count = MAX_TOUCH_POINTS;

        var i: i32 = 0;
        while (i < t_count) : (i += 1) {
            touch_points[@as(usize, @intCast(i))] = rl.getTouchPosition(i);
        }

        // Draw
        //----------------------------------------------------------------------------------
        rl.beginDrawing();
        defer rl.endDrawing();

        rl.clearBackground(rl.Color.white);

        for (0..@as(usize, @intCast(t_count))) |j| {
            if ((touch_points[j].x > 0) and (touch_points[j].y > 0)) {
                rl.drawCircleV(touch_points[j], 34, rl.Color.orange);
                rl.drawText("Touch", @as(i32, @intFromFloat(touch_points[j].x)) - 10, @as(i32, @intFromFloat(touch_points[j].y)) - 10, 40, rl.Color.black);
            }
        }
        //----------------------------------------------------------------------------------
    }
}
```

Is there any difference for this binding, compared to the original C version? Anything I have done wrong in the example?

**Comment by Not-Nik** [comment]
Not entirely sure if GLFW supports multitouch. Also Zenbooks might not bes supported. Can you try out the original raylib example?

**Comment by Logickin-Lambda** [comment]
> Not entirely sure if GLFW supports multitouch. Also Zenbooks might not bes supported. Can you try out the original raylib example?

Yep! In fact, I did tried the original example on the website to confirm that raylib works without any issue before I decided to submit the issue; as you can see, I have included [this link](https://www.raylib.com/examples/core/loader.html?name=core_input_multitouch) from the previous post.

![image](https://github.com/user-attachments/assets/f88b323f-9fb6-4155-a2b3-e388ff423833)

I have captured my screen from the example above, showing that in theory, multitouch should works with zenbook; however, with our current binding, it seems to stuck at zero. Perhaps once I have some time, I will try if the original c library behave the same or differently.

**Comment by Not-Nik** [comment]
The web example doesn't use GLFW, so if you find the time, please test it again with a native C build.

**Comment by dragsbruh** [comment]
I tried the web example on my touchscreen laptop and it does register multiple touches. However, when I built the example it does not recognize a single touch.

**Comment by Logickin-Lambda** [comment]
> I tried the web example on my touchscreen laptop and it does register multiple touches.

Could you show me your working solution? There might be a chance I got messed up with my code, so I wish to see the difference. I also wrote that in zig based on the web version, but I didn't seem to get any number with the function.

**Comment by dragsbruh** [comment]
I apologize for any miscommunication on my part but I mean I tried to build the source code in web example natively to an exe, and multiple touches don't seem to work there, but the web example showcased on raylib.com seems to work fine for me and detects multiple touches but the native build doesn't even detect any touch.

**Comment by Not-Nik** [comment]
That seems to suggest that GLFW just doesn't support some touchscreen laptops, which isn't something raylib-zig or even raylib can fix.
